packages/perspective/src/translators/swap-claim-view.ts

```diff
@@ -2,7 +2,7 @@ import { Translator } from './types.js';
 import {
   SwapClaimView,
   SwapClaimView_Opaque,
-} from '@buf/penumbra-zone_penumbra.bufbuild_es/penumbra/core/component/dex/v1/dex_pb.js';
+} from '@penumbra-zone/protobuf/penumbra/core/component/dex/v1/dex_pb';
```

Comment:
question: any chance in the future we could simplify the imports? Maybe after applying v2 of the generator we could export types as barrels?

Reply:
the service definitions have a barrel export defined, but i think trying to collect other types into barrels is likely to be very complex and not very useful.